db/create_tables.sql

```diff
@@ -39,7 +39,7 @@ CREATE TABLE IF NOT EXISTS `events` (
   `type` VARCHAR(100) NOT NULL,
   `status` VARCHAR(100) DEFAULT NULL,
   `state_machine_instance_id` VARCHAR(255) NOT NULL,
-  `data` LONGTEXT,
+  `event_data` BLOB,
   `event_source` VARCHAR(100) DEFAULT NULL,
   `created_at` DATETIME(3) DEFAULT NULL,
   `updated_at` DATETIME(3) DEFAULT NULL,
```

Comment:
same as above. We may need it, if are planning to change the status of the event.